operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -77,6 +77,8 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception are educational guides, as first person writing helps them to keep the reader interested in  the subject.
```

Comment:
```suggestion
An exception is Customer Education content which is written in first-person to align more closely with the subject matter.
```
I'm 2/5 on this exact wording but I'm inclined toward either a more substantial reason or no reason at all.

Reply:
I love your wording! To me keeping the reader engaged is the most important part of a CE guide. If I want to educate someone on a matter by using a text, I first of all have to make sure they read it.

How about a mix like 

```suggestion
An exception is Customer Education content which can be written in first-person to keep the audience focused and interested.
```

?

Reply:
I understand that. My only change is:
"An exception is Customer Education content which is written in first-person to keep the audience focused and interested."

As an aside, I am wary of stating that first-person keeps the audience focused and interested as first-person is more of a tool used in fiction writing than technical content. That said, my perspective is biased and comes from the opposite end of the spectrum and so is just an opinion/comment (especially as I don't know much about sales/marketing writing). :)

Reply:
Wouldn't that make it impossible to write educational content from second person?

Reply:
I didn't realize that was an option, so in that case "can be" would be more appropriate. Perhaps we can say "can be written in first or second person"?